r/rsparkling/rsparkling.Rproj

```diff
@@ -13,4 +13,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
-PackageRoxygenize: rd,collate,namespace
+PackageRoxygenize: rd,collate
```

Comment:
I wouldn't keep it, @erin @navdeep-G do you need to keep `rsparkling.Rpoj` in git history?